Warning: no access to tty (Bad file descriptor).
Thus no job control in this shell.
/var/spool/PBS/mom_priv/jobs/175657.hpcc-con.SC: line 7: module: command not found
/var/spool/PBS/mom_priv/jobs/175657.hpcc-con.SC: line 8: conda: command not found
/var/spool/PBS/mom_priv/jobs/175657.hpcc-con.SC: line 9: activate: No such file or directory
/home/s1510756/py27/lib/python2.7/site-packages/pip/commands/install.py:194: UserWarning: Disabling all use of wheels due to the use of --build-options / --global-options / --install-options.
  cmdoptions.check_install_build_global(options)
Collecting matplotlib==1.5.1 (from -r /home/s1510756/hpcc/requirements.txt (line 1))
  Using cached matplotlib-1.5.1.tar.gz
Collecting numpy==1.10.4 (from -r /home/s1510756/hpcc/requirements.txt (line 2))
  Downloading numpy-1.10.4.zip (4.6MB)
Collecting scikit-learn==0.17.1 (from -r /home/s1510756/hpcc/requirements.txt (line 3))
  Downloading scikit-learn-0.17.1.tar.gz (7.9MB)
Collecting scipy==0.17.0 (from -r /home/s1510756/hpcc/requirements.txt (line 4))
  Downloading scipy-0.17.0.tar.gz (12.4MB)
Collecting unidiff==0.5.2 (from -r /home/s1510756/hpcc/requirements.txt (line 5))
  Downloading unidiff-0.5.2.tar.gz
Collecting enum34==1.1.2 (from -r /home/s1510756/hpcc/requirements.txt (line 6))
  Using cached enum34-1.1.2.tar.gz
Collecting python-dateutil (from matplotlib==1.5.1->-r /home/s1510756/hpcc/requirements.txt (line 1))
  Downloading python-dateutil-2.6.0.tar.gz (258kB)
Collecting pytz (from matplotlib==1.5.1->-r /home/s1510756/hpcc/requirements.txt (line 1))
  Downloading pytz-2016.10.tar.gz (287kB)
Collecting cycler (from matplotlib==1.5.1->-r /home/s1510756/hpcc/requirements.txt (line 1))
  Downloading cycler-0.10.0.tar.gz
Collecting pyparsing!=2.0.0,!=2.0.4,>=1.5.6 (from matplotlib==1.5.1->-r /home/s1510756/hpcc/requirements.txt (line 1))
  Downloading pyparsing-2.1.10.zip (1.4MB)
Collecting six>=1.5 (from python-dateutil->matplotlib==1.5.1->-r /home/s1510756/hpcc/requirements.txt (line 1))
  Downloading six-1.10.0.tar.gz
Skipping bdist_wheel for matplotlib, due to binaries being disabled for it.
Skipping bdist_wheel for numpy, due to binaries being disabled for it.
Skipping bdist_wheel for scikit-learn, due to binaries being disabled for it.
Skipping bdist_wheel for scipy, due to binaries being disabled for it.
Skipping bdist_wheel for unidiff, due to binaries being disabled for it.
Skipping bdist_wheel for enum34, due to binaries being disabled for it.
Skipping bdist_wheel for python-dateutil, due to binaries being disabled for it.
Skipping bdist_wheel for pytz, due to binaries being disabled for it.
Skipping bdist_wheel for cycler, due to binaries being disabled for it.
Skipping bdist_wheel for pyparsing, due to binaries being disabled for it.
Skipping bdist_wheel for six, due to binaries being disabled for it.
Installing collected packages: numpy, six, python-dateutil, pytz, cycler, pyparsing, matplotlib, scikit-learn, scipy, unidiff, enum34
  Running setup.py install for numpy: started
    Running setup.py install for numpy: still running...
    Running setup.py install for numpy: still running...
    Running setup.py install for numpy: finished with status 'done'
  Running setup.py install for six: started
    Running setup.py install for six: finished with status 'done'
  Running setup.py install for python-dateutil: started
    Running setup.py install for python-dateutil: finished with status 'done'
  Running setup.py install for pytz: started
    Running setup.py install for pytz: finished with status 'done'
  Running setup.py install for cycler: started
    Running setup.py install for cycler: finished with status 'done'
  Running setup.py install for pyparsing: started
    Running setup.py install for pyparsing: finished with status 'done'
  Running setup.py install for matplotlib: started
    Running setup.py install for matplotlib: finished with status 'done'
  Running setup.py install for scikit-learn: started
    Running setup.py install for scikit-learn: finished with status 'error'
    Complete output from command /home/s1510756/py27/bin/python -u -c "import setuptools, tokenize;__file__='/var/tmp/pbs.175657.hpcc-con/pip-build-VUuexD/scikit-learn/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" build_ext "-L /home/s1510756/lib64 -I /home/s1510756/lib/libffi-3.2.1/include" install --record /var/tmp/pbs.175657.hpcc-con/pip-ty7_Sz-record/install-record.txt --single-version-externally-managed --compile:
    Partial import of sklearn during the build process.
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/var/tmp/pbs.175657.hpcc-con/pip-build-VUuexD/scikit-learn/setup.py", line 247, in <module>
        setup_package()
      File "/var/tmp/pbs.175657.hpcc-con/pip-build-VUuexD/scikit-learn/setup.py", line 237, in setup_package
        .format(scipy_req_str, instructions))
    ImportError: Scientific Python (SciPy) is not installed.
    scikit-learn requires SciPy >= 0.9.
    Installation instructions are available on the scikit-learn website: http://scikit-learn.org/stable/install.html
    
    
    ----------------------------------------
Command "/home/s1510756/py27/bin/python -u -c "import setuptools, tokenize;__file__='/var/tmp/pbs.175657.hpcc-con/pip-build-VUuexD/scikit-learn/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" build_ext "-L /home/s1510756/lib64 -I /home/s1510756/lib/libffi-3.2.1/include" install --record /var/tmp/pbs.175657.hpcc-con/pip-ty7_Sz-record/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /var/tmp/pbs.175657.hpcc-con/pip-build-VUuexD/scikit-learn/
/home/s1510756/py27/lib/python2.7/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment.
  warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.')
Traceback (most recent call last):
  File "/home/s1510756/hpcc/vcc-combine.py", line 2, in <module>
    from sklearn.metrics import precision_recall_curve
ImportError: No module named sklearn.metrics
